build/NuSpecs/ProjectReunion-Nuget-Native.C.props

```diff
@@ -20,10 +16,21 @@
     <Link>
       <AdditionalDependencies>
         $(MSBuildThisFileDirectory)..\..\lib\win10-$(_ProjectReunionFoundationPlatform)\Microsoft.ProjectReunion.lib;
+        %(AdditionalDependencies);
+      </AdditionalDependencies>
+    </Link>
+  </ItemDefinitionGroup>
+
+  <ItemDefinitionGroup Condition="'$(AppxPackage)' != 'true'">
+    <Link>
+      <AdditionalDependencies>
         $(MSBuildThisFileDirectory)..\..\lib\win10-$(_ProjectReunionFoundationPlatform)\Microsoft.ProjectReunion.Bootstrap.lib;
         %(AdditionalDependencies);
       </AdditionalDependencies>
     </Link>
+    <PostBuildEvent>
+      <Command>xcopy /y "$(MSBuildThisFileDirectory)..\..\runtimes\lib\native\$(_ProjectReunionFoundationPlatform)\Microsoft.ProjectReunion.Bootstrap.dll" "$(OutDir)"</Command>
```

Comment:
Is this inside the $(OutDir)/buildoutput?  We really should avoid this type of relative reference.  Relative file paths are terrible for refactoring type of changes which are typical this early in a repo's life.  Please file an issue to fix this later.  I don't want to block this change for such a thing.

Reply:
It's self-relative.

MSBuildThisFileDirectory is literally this file - ProjectReunion-Nuget-Native.C.props. The relative path'ing is to back us up past build/Nuspecs so we can drill down into runtimes\...

If you know a $(TheRootOfThePackageContainingThisFile) instead that'd be better but when in Rome...